tests/Functional/ContentCallForTendersRenderTest.php

```diff
@@ -79,163 +65,240 @@ public function testTenderRendering(): void {
-      $this->assertEquals($labels_data[$index], $element->getText());
-    }
-    $values = $field_list[0]->findAll('css', 'dd.ecl-description-list__definition');
-    $this->assertCount(6, $values);
-    $values_data = [
-      'Open',
-      'Call for tenders reference',
-      '15 April 2020',
-      '30 April 2020',
-      "11 June $next_year, 09:30 (AEST)",
-      'Audit Board of the European Communities',
+    $details_html = $content_items[0]->getHtml();
+    $field_list_assert->assertPattern($details_expected_values, $details_html);
+    $field_list_assert->assertVariant('horizontal', $details_html);
+    $this->assertDeadlineDateStrike($content);
+
+    // Assert Introduction field.
+    $node->set('oe_summary', 'Call for tenders introduction')->save();
+    $this->drupalGet($node->toUrl());
+    $page_header_expected_values = [
+      'title' => 'Test Call for tenders node',
+      'description' => 'Call for tenders introduction',
+      'meta' => 'Call for tenders',
     ];
-    foreach ($values_data as $index => $value) {
-      $this->assertEquals($value, $values[$index]->getText());
-    }
+    $page_header_assert->assertPattern($page_header_expected_values, $page_header->getOuterHtml());
 
-    // Assert header of second field group.
-    $this->assertContentHeader($content_items[1], 'Description', 'description');
+    // Assert "Open" status.
+    $node->set('oe_call_tenders_opening_date', ['value' => $opening_date->format('Y-m-d')])->save();
+    $this->drupalGet($node->toUrl());
 
-    // Assert content of second field group.
-    $content_second_group = $content_items[1]->find('css', '.ecl-editor p');
-    $this->assertEquals('Call for tenders body', $content_second_group->getText());
+    $page_header_expected_values['meta'] = 'Call for tenders | Open';
+    $page_header_assert->assertPattern($page_header_expected_values, $page_header->getOuterHtml());
 
-    // Assert header of third field group.
-    $this->assertContentHeader($content_items[2], 'Documents', 'documents');
-    $this->assertMediaDocumentDefaultRender($content_items['2'], 'call_for_tenders_document');
+    $details_expected_values['items'] = [
+      [
+        'label' => 'Status',
+        'body' => 'Open',
+      ], [
+        'label' => 'Publication date',
+        'body' => '12 February 2020',
+      ], [
+        'label' => 'Opening date',
+        'body' => '14 February 2020',
+      ], [
+        'label' => 'Deadline date',
+        'body' => '21 February 2020, 01:00 (AEDT)',
+      ],
+    ];
+    $field_list_assert->assertPattern($details_expected_values, $content_items[0]->getHtml());
+    $this->assertDeadlineDateStrike($content);
 
-    // Assert Responsible department field label.
+    // Assert "Upcoming" status.
+    $opening_date = (clone $static_time)->modify('+ 2 days');
+    $node->set('oe_call_tenders_opening_date', ['value' => $opening_date->format('Y-m-d')])->save();
+    $this->drupalGet($node->toUrl());
```

Comment:
Same as per event, we need to move the time, run cron and refresh the page, not change the node, or else we are not testing our time based tags, since resaving the node will invalidate the tags anyway.